internal/metamorph/processor.go

```diff
@@ -615,6 +628,46 @@ func (p *Processor) ProcessTransaction(ctx context.Context, req *ProcessorReques
 	}
 }
 
+func (p *Processor) BulkProcessTransaction(reqs []*ProcessorRequest) {
```

Comment:
I'm not sure about this, but maybe we could change the name of this function to simply `ProcessTransactions`? Since we already have `ProcessTransaction`, it would be consistent with other methods in Arc, such as:

`SubmitTransaction`
`SubmitTransactions`
`PutTransaction`
`PutTransactions`